src/main/java/com/wolox/servingwebcontent/models/Book.java

```diff
@@ -0,0 +1,138 @@
+package com.wolox.servingwebcontent.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
```

Comment:
JPA needs an empty constructor. It won't be able to instantiate this class after finding the information in the DB

Reply:
so i should remove it right ?